autoload/vista/executive/nvim_lsp.vim

```diff
@@ -0,0 +1,70 @@
+" Copyright (c) 2019 Alvaro Muñoz
+" MIT License
+" vim: ts=2 sw=2 sts=2 et
+
+let s:provider = fnamemodify(expand('<sfile>'), ':t:r')
+
+let g:vista_executive_nvim_lsp_reload_only = v:false
```

Comment:
I'm guessing changing to the global variable is due to neovim lua does not have `vim.s.foo` support?

Reply:
correct, we can access ot for read but not change them from lua closure